man/org.freedesktop.systemd1.xml

```diff
@@ -1073,27 +1075,27 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
       last watchdog ping received from the service, or 0 if none was ever received.</para>
 
       <para><varname>ExecStartPre</varname>, <varname>ExecStart</varname>, <varname>ExecStartPost</varname>,
-      <varname>ExecReload</varname>, <varname>ExecStop</varname>, <varname>ExecStop</varname> each are arrays
-      of structures each containing: the binary path to execute; an array with all arguments to pass to the
-      executed command, starting with argument 0; a boolean whether it should be considered a failure if the
-      process exits uncleanly; two pairs of
+      <varname>ExecReload</varname>, <varname>ExecStop</varname>, and <varname>ExecStop</varname> are arrays
+      of structures where each struct contains: the binary path to execute; an array with all arguments to
+      pass to the executed command, starting with argument 0; a boolean whether it should be considered a
+      failure if the process exits uncleanly; two pairs of
       <constant>CLOCK_REALTIME</constant>/<constant>CLOCK_MONOTONIC</constant> microsecond timestamps when
       the process began and finished running the last time, or 0 if it never ran or never finished running;
       the PID of the process, or 0 if it has not run yet; the exit code and status of the last run. This
       field hence maps more or less to the corresponding setting in the service unit file but is augmented
       with runtime data.</para>
 
       <para><varname>LimitCPU</varname> (and related properties) map more or less directly to the
-      corresponding settings in the service unit files, however are set to 18446744073709551615 (i.e. -1) if
-      they aren't set.</para>
+      corresponding settings in the service unit files However, these are set to 18446744073709551615
+      (i.e. -1) if they aren't set.</para>
 
       <para><varname>Capabilities</varname> contains the configured capabilities, as formatted with
       <citerefentry project='man-pages'><refentrytitle>cap_to_text</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
       </para>
 
       <para><varname>SecureBits</varname>, <varname>CapabilityBoundingSet</varname>,
       <varname>MountFlags</varname> also correspond to the configured settings of the unit files, but are
-      encoded as the actual binary flag fields they are, rather than formatted as string.</para>
+      encoded as the actual binary flag fields they are rather than formatted as strings.</para>
```

Comment:
The original syntax is a bit weird, but without the comma it is harder to understand.

Reply:
Went with "of the unit files, but instead of being formatted as strings, they are encoded as the actual binary flags they are."